docs/reference/accessibility/orca-preferences/orca-preferences-braille.md

```diff
@@ -0,0 +1,75 @@
+
+# Braille preferences
+
+## Enable Braille Support
+
+Toggles braille display integration. Orca handles BrlTTY absence gracefully.  
+**Default value:** `Not checked`
```

Comment:
```suggestion

**Default value:** Not checked
```

Since it's not a code string. I think this is accurate across the preferences pages.